### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by bethpresten**
Added navbar with working links and pic, header with fonts, subtitle, WIP: input field

### CODE REVIEW

File: public/assets/css/style.css [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
+  margin-left: auto;
+  margin-right: auto;
+}
+
+#search {
+  width: 375px;
+  height: 50px;
+}
 
 body {
```
Comment (atlsykes) [inline public/assets/css/style.css:25]: You've got 2 body elements referenced in this file, please consolidate.

File: views/index.handlebars [hunk 2]
```diff
@@ -1 +1,73 @@
...
+<img id="multiple-runners" src="/assets/images/multiple-runners.png" alt="stick figure runners"/>
+</div>
+<br>
+<div class="field">
+  <label class="label is-large">City</label>
+  <div class="control has-icons-right">
+    <input class="input is-large" type="input" placeholder="Search" id="search">
+    <span class="icon is-small is-right">
+      <i class="fas fa-search"></i>
+    </span>
```
Comment (atlsykes) [inline views/index.handlebars:71]: can you add a 'submit' or 'search' button here?

Comment (bethpresten) [reply views/index.handlebars:71]: The search button is in process.  Our wireframe just had the magnifying glass.  Right now it's pushed all the way right.  I was WIP with it being pushed to the left, so it's there-ish.

File: public/assets/css/style.css (hunk 1 above)
Comment (bethpresten) [reply public/assets/css/style.css:25]: I consolidated on my end.

**Pull request closed without merging**